confgenerator/logging_processors.go

```diff
@@ -229,3 +231,26 @@ var LegacyBuiltinProcessors = map[string]LoggingProcessor{
 		},
 	},
 }
+
+// A LoggingProcessorExcludeLogs filters out logs according to a pattern.
+type LoggingProcessorExcludeLogs struct {
+	ConfigComponent `yaml:",inline"`
+	Condition       string `yaml:"condition" validate:"required,filter"`
+}
+
+func (r LoggingProcessorExcludeLogs) Type() string {
+	return "exclude_logs"
+}
+
+func (p LoggingProcessorExcludeLogs) Components(tag, uid string) []fluentbit.Component {
+	fil, err := filter.NewFilter(p.Condition)
```

Comment:
All that setup work, for a tiny little processor like this! :)

I guess we decided to not offer multiple filters or inclusions for now?

Reply:
Multiple filters still needs more discussion on how they should behave (AND semantics? OR? something else?); maybe in a future iteration.

Inclusion would be trivial to add though, since I'm fairly certain it's just swapping `Exclude` for `Regex` in the single `grep` filter at the end. I'd just have to write the tests. CC @qingling128 should we go ahead and add inclusion as well?

Reply:
IMO multiple filters has clear logic - both should be ORed (i.e., the first matching filter applies). I think it's only ambiguous if you have both include and exclude filters in the same list, which I'm not suggesting here.

Reply:
Resolved per the `match_any` change.